server/endpoint/healthz/endpoint.go

```diff
@@ -0,0 +1,94 @@
+package healthz
```

Comment:
All this should not be necessary. Check https://github.com/giantswarm/kvm-operator/blob/ebab431ef35478a4a7efc6830564a3e47f38e6a9/server/endpoint/endpoint.go.

Reply:
Ack. I based this on aws-operator which has the old structure. I'll update once this is merged.